### PR TITLE
[All] Reformat using go 1.19 fmt

### PR DESCRIPTION
**Pull request by SaveTheRbtz**
No functional changes.

Go 1.19 has a [whole new set of formatting rules](https://go.dev/doc/comment#syntax) that involve:
1. pre-formatted text
2. item lists
3. numbered lists
4. headings

This diff is basically `goimports -w -local github.com/onflow/flow-go` + a lot of manual fixes for the formatting.

PS. Mixing lists and preformatted text (e.g. ASCII graphics) is not iincompatible, used `<TAB> .` as a workaround.
PPS. Full design proposal for the new format for those who are interested can be found here: https://github.com/golang/proposal/blob/master/design/51082-godocfmt.md

### CODE REVIEW

**Comment by AlexHentschel** [comment]
thanks for going through all of this code and fixing formatting problems 🎖️

File: utils/unittest/epoch_builder.go [hunk 1]
```diff
@@ -111,15 +111,16 @@ func (builder *EpochBuilder) EpochHeights(counter uint64) (*EpochHeights, bool)
...
-//                 |                                                                                               |
-//     P                 A               B               C               D             E             F           G
-// +------------+  +------------+  +-----------+  +-----------+  +----------+  +----------+  +----------+----------+
-// | ER(P-1)    |->| ER(P)      |->| ER(A)     |->| ER(B)     |->| ER(C)    |->| ER(D)    |->| ER(E)    | ER(F)    |
-// | S(ER(P-2)) |  | S(ER(P-1)) |  | S(ER(P))  |  | S(ER(A))  |  | S(ER(B)) |  | S(ER(C)) |  | S(ER(D)) | S(ER(E)) |
-// +------------+  +------------+  +-----------+  +-----------+  +----------+  +----------+  +----------+----------+
-//                                                                             |                        |
-//                                                                             Setup                    Commit
+//	            |                                  EPOCH N                                                      |
+//	            |                                                                                               |
```
Comment (AlexHentschel) [inline utils/unittest/epoch_builder.go:115]: I think we lost some spaces here:
```suggestion
//	                |                                  EPOCH N                                                      |
//	                |                                                                                               |
```

**Review by AlexHentschel — APPROVED** [review]
Thanks for grinding through this. 
Suggested some minor revisions, to avoid loosing structure with the new formatting

**Comment by SaveTheRbtz** [comment]
merged Alex suggestions, squashed, rebased, re-applied the lint.

**Review by pattyshack — APPROVED** [review]

**Review by Kay-Zee — APPROVED** [review]

**Comment by SaveTheRbtz** [comment]
bors merge

**Comment by SaveTheRbtz** [comment]
bors retry

**Pull request merged**